wcivf/apps/elections/templates/elections/includes/_how-to-vote.html

```diff
@@ -1,61 +1,105 @@
 {% load humanize %}
-{% comment %}
+{% load i18n %}
 
+{% comment %}
 Voting system is currently one of:
 
 AMS: Additional Member System
 FPTP: First-past-the-post
 sv: Supplementary Vote
 STV: Single Transferable Vote
-PR-CL: Closed List
+PR-CL: Closed List 
+{% endcomment %}
 
-{%  endcomment %}
 <ul class="ds-details">
     <li style="list-style: none;">
         <details>
-            <summary>How you vote</summary>
+            <summary>{% trans 'How you vote' %}</summary>
             <p>
                 {% if voting_system.get_absolute_url %}
-                    This election uses <a href="{{ voting_system.get_absolute_url }}">{{ voting_system.name }}</a>.
+                    {% blocktrans trimmed with voting_system_url=voting_system.get_absolute_url %}
+                        This election uses <a href="{{ voting_system_url }}">{{ voting_system.name }}</a>.
```

Comment:
`voting_system.name` will fail here I think?

Reply:
Fixed in last fixup commit.